lib/request.js

```diff
@@ -10,7 +10,7 @@ var AzureRequest = function ( config ) {
 AzureRequest.prototype.buildConfig = function ( config ) {
 
   this.config          = config || {}
-  this.config.mediaURI = "https://media.windows.net/API/"
+  this.config.mediaURI = "https://media.windows.net/api/"
```

Comment:
I don't think this is correct. Docs maintain an uppercase API.

https://msdn.microsoft.com/en-us/library/azure/hh973617.aspx